ihatemoney/web.py

```diff
@@ -895,12 +896,23 @@ def strip_ip_addresses():
 @main.route("/<project_id>/statistics")
 def statistics():
     """Compute what each member has paid and spent and display it"""
-    today = datetime.now()
+    if g.project.has_bills():
+        first_date = g.project.get_least_recent_bill().date
+        last_date = g.project.get_most_recent_bill().date
+    else:
+        first_date = datetime.now() - relativedelta(months=12)
+        last_date = datetime.now()
+    # Infinite iterator towards the past
+    all_months = (last_date - relativedelta(months=i) for i in itertools.count())
+    # Stop when reaching one month before the first date
+    months = itertools.takewhile(
```

Comment:
Should we name this `active_months`?

Reply:
They are not all active (I explained this in the commit message).